src/components/CustomCollection/CustomCollection.module.scss

```diff
@@ -43,7 +43,8 @@
     box-sizing: border-box;
     border-radius: 4px;
     margin-top: 0;
-    font-family: 'Trebuchet MS';
+    font-family: 'Trebuchet MS', Lucida Grande, Lucida Sans Unicode, Lucida Sans,
```

Comment:
Most computers have trebuchet on them, but it's still a good practice to provide a font stack. In our css modules it will work for both cases. In storybook I loaded in a web font version of trebuchet because happo needs it. If I didn't obtain woff files for it, it would still look right if we run the app or storybook locally, because trebuchet is pre installed on pretty much all Mac and windows computers

The only case where the web fonts are needed is happo. I'm not so pressed about performance and asset loading on storybook as I am with the app itself. I made sure this does not cause the trebuchet files are not loaded in the portal.